src/tigerbeetle/main.zig

```diff
@@ -154,8 +155,21 @@ const Command = struct {
         }
 
         const grid_cache_size = @as(u64, args.cache_grid_blocks) * constants.block_size;
+
+        // The amount of bytes in `--cache-grid` must be a multiple of
+        // `constants.block_size` and `SetAssociativeCache.value_count_max_multiple`,
+        // and it may be converted to zero if a smaller value is passed in.
+        if (grid_cache_size == 0) {
+            fatal("Grid cache must be greater than {}MB. See --cache-grid", .{
+                @divExact(
+                    constants.block_size * Grid.Cache.value_count_max_multiple,
```

Comment:
Maybe extract this into `grid_cache_size_min` (since we have `grid_cache_size` and `grid_cache_size_warn`). Then the `@divExact` will fit on one line, too.

And if `grid_cache_size != 0`, then we can assert that `grid_cache_size >= grid_cache_size_min`.